src/containers/saved-page/components/mobile/SavedPage.tsx

```diff
@@ -265,6 +273,16 @@ const PlaylistCardLineup = ({
   playlistUpdates: number[]
   updatePlaylistLastViewedAt: (playlistId: number) => void
 }) => {
+  const { isLoaded, isEnabled } = useFlag(FeatureFlags.PLAYLIST_UPDATES_ENABLED)
```

Comment:
Don't think we need this whole song and dance - isEnabled *should* default to the default value if we're not loaded (double check me here), and I don't think we need a useEffect and useState, we should be able to just directly use isEnabled or possibly some combination like `const shouldShow = (isLoaded && isEnabled)`. But, test it. Sorry for the flag code still being a little rough, @raymondjacobson was just pointing out today that we may have some issues around default values with flags.

Reply:
yeah, i think @piazzatron is correct here -- all we need to do is check for isLoaded && isEnabled.

The one caveat around defaults is interesting. Just important to note that default isn't what's used, basically ever. Default only is what's used if there's an error loading the optimizely config, regardless of whether the flag is even defined in optimizely.

Reply:
Yep you are correct sir! Ah so much simpler thanks

Reply:
We don't even need to check for isLoaded. Just isEnabled

Reply:
Hmm just saw your comment about defaults. I think we're still good here since we have the `flagDefaults` that we set that is used in `getFeatureEnabled`

Reply:
Yeah -- i think we're good here. the only thing to note is that defaults should probably always be false and really dont have much meaning when they are true, we should probably change the others that are set to true to false, and then just make sure to delete the flag entirely after the feature is launched